packages/flutter/lib/src/material/chip.dart

```diff
@@ -35,6 +35,9 @@ const TextStyle _kLabelStyle = const TextStyle(
 ///  * [CircleAvatar]
 ///  * <https://www.google.com/design/spec/components/chips.html>
 class Chip extends StatelessWidget {
+  /// Creates a material design chip.
+  ///
+  ///  * [onDeleted] determines whether the chip has a delete button.
```

Comment:
This also gets called when the delete button is pressed, so maybe we should state the obvious. Something like:

If specified the chip has a delete button. This callback runs when the delete button is pressed.